docs/self-hosted/oel/oauth2/token-prefix.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: token-prefix
+title: Custom OAuth2 Token Prefix
+sidebar_label: Custom Token Prefix
+---
+
+# Configurable Token Prefix for OAuth2 Tokens
```

Comment:
Do you want to override the `title` here? 
Usually you don't set a H1 since its already generated through `title`

```suggestion
```

Reply:
Ah, sorry, yes this was intentional for SEO